Chat/File.cpp

```diff
@@ -43,6 +43,11 @@ bool File::SetFile(const std::string &path)
                              FILE_ATTRIBUTE_NORMAL,     //nothing-specific-file
                              NULL);
     //bytes divided by kilobyte size
+    if(file == NULL)
```

Comment:
msdn says that failure case return value is NULL
https://msdn.microsoft.com/en-us/library/windows/desktop/ms682411(v=vs.85).aspx